Makefile

```diff
@@ -32,10 +33,15 @@ cross: clean
 .PHONY: vendor
 vendor: vendor.conf
 	@vndr 2> /dev/null
-	@script/validate/check-git-diff vendor
+	@scripts/validate/check-git-diff vendor
+
+## Generate man pages from go source and markdown
+.PHONY: manpages
+manpages:
+	@man/generate.sh
```

Comment:
Can you use wildcards in `Makefile` rules like that? I didn't think so, you would need [`$(wildcard man/man*)`](https://www.gnu.org/software/make/manual/html_node/Wildcard-Function.html), I think.

**But** be careful: this:
```
foo bar: baz
     make-foo-and-bar baz
```
Does not cause `make-foo-and-bar baz` to run once whenever `baz` changes. It will be run once for `foo` and once for `bar`. That is unlikely to be what you wanted, at least judging from the lack of parameters to `man/generate.sh` in your example (if it had been passed `$@` then it might be correct).